sapi/fpm/fpm/fpm_status.c

```diff
@@ -92,6 +97,9 @@ int fpm_status_handle_request(void) /* {{{ */
 			return 1;
 		}
 
+		struct fpm_scoreboard_proc_s procs[scoreboard_p->nprocs];
```

Comment:
I'm not sure if this is gonna work on all supported compilers. We also target C89 so definitions has to be at the beginning of block.

Reply:
Considering that VLA is C99, you can't really use it though.